frontend/package.json

```
@@ -9,19 +9,20 @@
     "lint": "next lint"
```

Jeg er fan av måten vi gjør det på i frontenden på ndla. Liker når det finnes en kommando for å formatere alle filer vi vil formatere uten å måtte inn der med en editor.

Hva tenker dere om noe lignende? 😄 
```suggestion
    "lint": "next lint && yarn format-check",
    "format": "prettier '(app/**/*|*)(*.js|*.jsx|*.ts|*.tsx|*.graphql|*.json)' --write",
    "format-check": "prettier '(app/**/*|*)(*.js|*.jsx|*.ts|*.tsx|*.graphql|*.json)' --check"
```

---

Jeg liker det. Og hadde noe lignende i tankene, kan like gjerne ta det inn også her.